1_assignment.js

```diff
@@ -0,0 +1,31 @@
+const item = [
+    { name: "Milk Bottle", price: 30, discountPercent: 10},
+    { name: "Cheese", price: 10, discountPercent: 0},
+    { name: "Packaged Meat (1kg)", price: 100, discountPercent: 5},
+  ]
+
+
+// METHOD WAY
+
+const calculatePrices = (itemArr) => {
+    let totalPriceArray = itemArr.map(item => item.price - (item.price * (item.discountPercent/100)));
+    let total = 0
+    let totalPrice = totalPriceArray.forEach(price => {
+        total += price;
+    });
+    console.log(total + " USD")
```

Comment:
The function is correct, but it should return the final price in string form such as "132 USD".

Reply:
alright Oh i get it :D